docs/src/building/guidance.md

```diff
@@ -138,3 +138,34 @@ for example, although this can run afoul of SELinux labeling.
 
 There is a dedicated document for [secrets](secrets.md),
 which is a special case of configuration.
+
+## Handling read-only vs writable locations
+
+The high level pattern for bootc systems is summarized again
+this way:
+
+- Put read-only data and executables in `/usr`
+- Put configuration files in `/usr` (if they're static), or `/etc` if they need to be machine-local
+- Put "data" (log files, databases, etc.) underneath `/var`
+
+However, some software installs to `/opt/<application>` and has
+all three types of data undernath its directory.  For example, it
+may write log files to `/opt/<application>/logs`.  A simple way to handle
+this is to change the directories that need to be writble to symbolic links
+to `/var`:
+
+```dockerfile
```

Comment:
small nit - inconsistent usage of `<application>` and `application`.  I probably wouldn't have even noticed this except the one is missing the closing `>` and it triggered my brain parser :laughing:

Reply:
Thanks!  Good catch.